code/game/objects/items/charter.dm

```diff
@@ -81,7 +81,7 @@
 
 /obj/item/station_charter/proc/rename_station(designation, uname, ureal_name, ukey)
 	set_station_name(designation)
-	minor_announce("[ureal_name] has designated your station as [html_decode(station_name())]", "Captain's Charter") //decode station_name to avoid minor_announce double encode
+	minor_announce("[ureal_name] istasyonun adını [html_decode(station_name())] olarak belirledi.", "Kaptanın Talimatı") //decode station_name to avoid minor_announce double encode
```

Comment:
```suggestion
	minor_announce("[ureal_name] istasyonun adını [html_decode(station_name())] olarak belirledi.", "Kaptanın Fermanı") //decode station_name to avoid minor_announce double encode
```